index.html

```diff
@@ -171,8 +171,23 @@ <h2 class="About" >About Myself </h2>
 </ul>
 <hr>
 </div>
+</section>
 <!-- the contact form's section -->
 
+<section class="form">
+  <div class="contact-sec">
+    <h2>Contact me</h2>
+     <p>If you have an application you <br> are interested in developing,a <br> feature that you need built or a <br> project that needs coding.I'd <br> love to help with it.</p>
+  </div>
+      <form action="https://formspree.io/f/mzbowdel" method="post">
+        <ul>
+          <li><input type="text" class="name" id="name" maxlength="30" placeholder="Tshepo Ishimwe" name="name" required></li>
+          <li><input type="email" class="email"  placeholder="tshepomakama45@gmail.com" id="email" name="email" required></li>
+          <li><textarea name="message" class="msg"  id="msg" maxlength="500" placeholder="Write your message here." required></textarea></li>
+          <li><button type="submit" class="btn" id="btn">Get in touch</button></li>
+        </ul>
+      </form>
```

Comment:
> [OPTIONAL] Since the previous code reviewer did not mention it, it would be great if you could double-check and make sure to align the spacing of the contact form more closely with the design provided in Figma. This attention to detail will help ensure consistency and a polished user experience. Keep up the good work! 👍
> 
> Figma design:
> 
> ![aa1](https://github.com/Tshepo45/Mobile-Finish/assets/40009354/8a8a74c5-9330-4604-9fcf-10727776eb3f)
> 
> Your design:
> 
> ![aa](https://github.com/Tshepo45/Mobile-Finish/assets/40009354/795113d9-1ab6-4615-8803-1717fe9ab0d4)
>